frontend/packages/kubevirt-plugin/src/utils/sort.ts

```diff
@@ -46,3 +47,17 @@ export const ignoreCaseSort = <T>(
   };
   return array.sort((a, b) => resolve(a).localeCompare(resolve(b)));
 };
+
+export const removeOSDups = (osArr: OperatingSystemArray): OperatingSystemArray => {
+  const osNames: { [key: string]: string } = {};
+  osArr.forEach((os) => {
+    if (osNames[os.name]) {
+      osNames[os.name] = os.id > osNames[os.name] ? os.id : osNames[os.name];
```

Comment:
what happens when the comparison misbehaves? - shouldn't we test at least for numbers?

Reply:
I checked the comparison with several examples - they all went OK.
@suomiy  can you think of a scenario that is would misbehave ?

Reply:
for example `"ubuntu9.10" > "ubuntu10.04"`